### DISCUSSION TITLE
Stopping a light show and restarting it

### DISCUSSION THREAD

**Discussion by worldpeace-germany** [missionpinball/missionpinball#128] [Q&A] [answered]
Hello,

let's assume I have light show running when my base mode starts, e.g. light 42 flashing green. That show should play at all times while the base mode is active. Now a switch is activated and that light should not flash green anymore but red based on a new given light show. Based on that event I can stop the light show which flashes the light green and I can start to flash it red based on this event. So far no problem. Now when the red light show is over I want to start the green again. It seems there is no event which is posted when a light show is over. How can I achieve this?

**Comment by BenTripoli** [comment, marked as answer]
Have a look at Priority in show or light player. Leave the green show running and the red show takes over with priority, when the red is finished,  green should reappear. 
https://missionpinball.org/config/show_player/
